src/Gizmos/planeRotationGizmo.ts

```diff
@@ -34,16 +35,19 @@ export class PlaneRotationGizmo extends Gizmo {
      */
     public onSnapObservable = new Observable<{ snapDistance: number }>();
 
+    private _isEnabled: boolean = true;
+    private _parent: RotationGizmo ;
+
     /**
      * Creates a PlaneRotationGizmo
      * @param gizmoLayer The utility layer the gizmo will be added to
      * @param planeNormal The normal of the plane which the gizmo will be able to rotate on
      * @param color The color of the gizmo
      * @param tessellation Amount of tessellation to be used when creating rotation circles
      */
-    constructor(planeNormal: Vector3, color: Color3 = Color3.Gray(), gizmoLayer: UtilityLayerRenderer = UtilityLayerRenderer.DefaultUtilityLayer, tessellation = 32) {
+    constructor(planeNormal: Vector3, color: Color3 = Color3.Gray(), gizmoLayer: UtilityLayerRenderer = UtilityLayerRenderer.DefaultUtilityLayer, tessellation = 32, parent: RotationGizmo) {
```

Comment:
does this need to be parent?: RotationGizmo as it is following paremeters that have defaults, this needs to be optional or have a default, everywhere the parent property has been added

Reply:
The original approach in the coding seemed to be controlling the attachedMesh only to active gizmos. I have maintained the same approach here as well.
In this change, I have added isEnabled for individual gizmos.
When you disable a Gizmo, attachedMesh is null and if you enable it immediately without a mesh change, I needed a way to find the correct mesh.
Hence the parent.